modules/apps/product-navigation/product-navigation-control-menu-web/src/main/resources/META-INF/resources/js/TabsContent.js

```diff
@@ -0,0 +1,178 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import classNames from 'classnames';
+import {fetch, objectToFormData} from 'frontend-js-web';
+import PropTypes from 'prop-types';
+import React, {useEffect, useMemo, useState} from 'react';
+
+import 'product-navigation-control-menu/css/TabsPanel.scss';
+
+import {normalizeContent} from './AddPanel';
+import {
+	useDisplayGridContext,
+	useGetContentsURLContext,
+	useNamespaceContext,
+} from './AddPanelContext';
+import Collapse from './Collapse';
+import ContentOptions from './ContentOptions';
+import SearchForm from './SearchForm';
+import SearchResultsPanel from './SearchResultPanel';
+import TabItem from './TabItem';
+
+const CONTENT_TAB_ID = 'content';
+const INITIAL_EXPANDED_ITEM_COLLECTIONS = 3;
+
+const TabsContent = ({tab}) => {
```

Comment:
There are several ways of declaring the components throughout the pull request:

- `const TabsContent = ({prop1}) => {}`
- `function TabsContent({prop1}) {}`
- `const TabsContent = (props) => {}`

I'm more inclined to option 2, but as we don't have a defined standard in the portal for this, It's up to you to choose one, but I'd stick to one for all the cases :)